apps/microsoft-teams/frontend/.nvmrc

```diff
@@ -0,0 +1 @@
+lts/hydrogen
```

Comment:
should we specify the explicit node dependency or the LTS? i don't have a strong opinion about it, but i notice in most places at contentful we do the version number

Reply:
Sounds good, updated to `v18.13.0`, which is the version used by several of our other apps.